lib/ServerFiles.php

```diff
@@ -199,4 +199,36 @@ public function deleteFile() {
 
 		return new Result();
 	}
+
+	/**
+	 * List file in the specified directory under the server root
```

Comment:
```suggestion
	 * List files in the specified directory under the server root
```
IMO it returns a list of all the files in the directory.